introduction.md

```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam augue risus, semper nec elit nec, dapibus fringilla eros. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Maecenas consequat ipsum sollicitudin enim ornare, id sollicitudin erat vehicula. Etiam auctor auctor consequat. Donec at auctor magna. Nulla sed lacinia metus. Pellentesque lobortis elementum enim. Mauris tincidunt sem eros, at condimentum erat tempor vel. Vestibulum pellentesque aliquam neque, a euismod mi ultrices vitae.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.